streams/core.py

```diff
@@ -105,6 +105,23 @@ def emit(self, x):
                 result.append(r)
         return [element for element in result if element is not None]
 
+    def update(self, x, who=None):
+        self.emit(x)
+
+    def connect(self, parent):
+        ''' Connect another child to stream.
+            Note that parents go downstream and children go upstream.
```

Comment:
Can we put this somewhere special? I think this may trip potential new devs up and thus should be placed somewhere they'd find it.